src/Builder/Builder.php

```diff
@@ -18,14 +18,13 @@ public function __toString(): string
     /**
      * @param array<string|int, string> $mapping
      */
-    protected function buildQuery(array $mapping, Closure $callback, string $q = ''): string
+    protected function buildQuery(array $mapping, Closure $callback, string $query = ''): string
     {
         foreach ($mapping as $parameter => $property) {
-            if (!isset($this->{$property})) {
+            if (!property_exists($this, $property)) {
```

Comment:
This will cause an exception when trying to access the uninitialized properties below

Reply:
Done.